tools/README.md

```diff
@@ -0,0 +1,110 @@
+Created config denier/istio/denyall at revision 881
+Created config checknothing/istio/denyrequest at revision 882
+Created config rule/istio/mixercachebuster at revision 883
+```
+The deployment is now complete. You can verify the deployment using standard `kubectl` commands:
+```
+$ kubectl get po --all-namespaces
+NAMESPACE      NAME                                                   READY     STATUS    RESTARTS   AGE
+fortio         fortio1-1966733334-xj5f6                               1/1       Running   0          8m
+fortio         fortio2-3044850348-v5f74                               1/1       Running   0          8m
+istio-system   istio-ca-1363003450-gvtmn                              1/1       Running   0          7m
+istio-system   istio-ingress-1732553340-gv41r                         1/1       Running   0          7m
+istio-system   istio-mixer-3192291716-psskv                           3/3       Running   0          8m
+istio-system   istio-pilot-3663920167-4ns3g                           2/2       Running   0          7m
+<SNIP>
+```
+You can now run the performance tests:
+```
+$ run_tests
+```
+
+The first test case uses the default loadbalancer and no Istio mesh or Istio Ingress Controller. The following command tells Fortio on the VM to run a load test against the Fortio echo server running in the Kubernetes cluster:
+```
+### Running: curl http://$VM_IP/fortio/?json=on&qps=-1&t=30s&c=48&load=Start&url=http://$K8S_FORTIO_EXT_IP:8080/echo
+```
+The following arguments are passed to the Fortio server running on the GCE VM:
+
+| Argument                                | Description                             |
+| --------------------------------------- | --------------------------------------- |
+| json=on                                 | Sets output in json format              |
+| qps=-1                                  | Requested queries per second to "max"   |
+| t=30s                                   | Requested duration to run load test     |
+| c=48                                    | Number of connections/goroutine/threads |
+| qps=-1                                  | Requested queries per second to "max"   |
+| load=Start                              | Tells Fortio to be a load generator     |
+| url=http://$K8S_FORTIO_EXT_IP:8080/echo | The target to load test                 |
+
+The second test case uses the Fortio Ingress with no Istio mesh and the same arguments as the first test:
+```
+### Running: curl http://$VM_IP/fortio/?json=on&qps=-1&t=30s&c=48&load=Start&url=http://$NON_ISTIO_INGRESS/echo
+```
+
+The third test case uses the Istio Ingress with the same arguments as the first test. This is the test that performs load testing
+of the Istio service mesh:
+```
+### Running: curl http://$VM_IP/fortio/?json=on&qps=-1&t=30s&c=48&load=Start&url=http://$ISTIO_INGRESS/fortio1/echo
+```
+
+Compare the test results to understand the load differential between the 3 test cases.
```

Comment:
maybe add using the web UI of fortio ?

Reply:
for instance:
```
 get_ips
+++ VM Ip is 35.199.49.6 - visit http://35.199.49.6/fortio/
+++ In k8s fortio external ip: http://35.199.53.5:8080/fortio/
+++ In k8s non istio ingress: http://35.190.84.121/fortio/
+++ In k8s istio ingress: http://35.199.45.122/fortio1/fortio/ and fortio2
```
so then go to  http://35.199.45.122/fortio1/fortio/ and talk to echosrv2.istio.svc.cluster.local:8080
and vice versa